lib/Cake/View/Helper/HtmlHelper.php

```diff
@@ -173,6 +173,10 @@ public function __construct(View $View, $settings = array()) {
 /**
  * Adds a link to the breadcrumbs array.
  *
+ * ### Options
+ *
+ * 'prepend' Prepend the breadcrumb to. Using this option
```

Comment:
"Using this option"?
There seems to be something missing.

Also maybe a dash here?

    'prepend' - Prepends the breadcrump instead of appending it.